src/main/scala/com/evolutiongaming/util/dispatchers/Instrumented.scala

```diff
@@ -69,18 +69,20 @@ object Instrumented {
     def metrics(id: String, name: String, registry: MetricRegistry): Instrument = {
       val queue = registry histogram s"$name.queue"
       val run = registry histogram s"$name.run"
-      val workers = registry counter s"$name.workers"
+      val workersCurrent = registry counter s"$name.workers.current" // current value, very volatile
+      val workers = registry histogram s"$name.workers"              // aggregated metrics
```

Comment:
hm, are you sure about changing type of metric?

Reply:
it will look like
workers
 current <- current value of workers
 ... 
 p50
 p99
 p999
 max
 count
 m1_rate
 m5_rate
...